presto-main/src/main/java/com/facebook/presto/server/CoordinatorModule.java

```diff
@@ -222,13 +218,9 @@ protected void setup(Binder binder)
         // node monitor
         binder.bind(ClusterSizeMonitor.class).in(Scopes.SINGLETON);
 
-        // statistics calculator
+        // stats and cost calculators
         binder.install(new StatsCalculatorModule());
-
-        // cost calculator
-        binder.bind(CostCalculator.class).to(CostCalculatorUsingExchanges.class).in(Scopes.SINGLETON);
-        binder.bind(CostCalculator.class).annotatedWith(EstimatedExchanges.class).to(CostCalculatorWithEstimatedExchanges.class).in(Scopes.SINGLETON);
-        binder.bind(CostComparator.class).in(Scopes.SINGLETON);
+        binder.install(new CostCalculatorModule());
```

Comment:
Does it make sense to have `StatsCalculatorModule` and `CostCalculatorModule` as separate modules?
Maybe one would be enough and more convenient?

Reply:
I'm dropping this commit. It is not needed for what i'm doing now.